### PR TITLE
feat: add cep formatter

### PR DESCRIPTION
**Pull request by rfoel**
I found the need to have a CEP formatter to get the data from APIs and display to the user. Some of the data have nothing but digits and others have a dot in them. This package should work for both cases.

### CODE REVIEW

File: src/utilities/index.ts [hunk 1]
```diff
@@ -1,8 +1,7 @@
 export { isValid as isValidPIS } from './pis';
-export { isValid as isValidCEP } from './cep';
 export { isValid as isValidPhone } from './phone';
 export { isValid as isValidEmail } from './email';
 export { format as formatJudicialProcess } from './judicial-process';
-export { format as formatBoleto, isValid as isValidBoleto } from './boleto';
```
Comment (hyanmandian) [inline src/utilities/index.ts]: I think you removed it by accident

**Comment by hyanmandian** [comment]
Awesome job @rfoel . Can you update the docs with your new utility?

**Comment by rfoel** [comment]
@hyanmandian I didn't any docs for formatters. Should I add it anyway?

**Comment by hyanmandian** [comment]
Yes, I didn't put others formatters cuz I didn't have time :(

**Comment by rfoel** [comment]
@hyanmandian Where will we add it? Under utilities? At the bottom, separating from validations, or right under its peer?

**Comment by hyanmandian** [comment]
@rfoel I think that you can write after the last utility. Another time I will improve the docs and group related utilities.

**Comment by rfoel** [comment]
@hyanmandian done :)

**Comment by hyanmandian** [comment]
@all-contributors please add @rfoel  for code, doc

**Pull request merged**